RealestateApi/Service/RealEstateService.cs

```diff
@@ -0,0 +1,21 @@
+﻿using RealEstateApi.Dto.Response;
+using RealEstateApi.Repository.Interfaces;
+using RealEstateApi.Service.Interfaces;
+
+namespace RealEstateApi.Service
+{
+    public class RealEstateService : IRealEstateService
+    {
+        private readonly IRealEstateRepository _realEstateRepository;
+
+        public RealEstateService(IRealEstateRepository realEstateRepository)
+        {
+            _realEstateRepository = realEstateRepository;
+        }
+
+        public async Task<RealEstateResponseDto> GetRealEstateAsync()
```

Comment:
Instead of returning RealEstateResponseDto you can delete that class and you can return: "Task<List<RealEstate>>"